python/sglang/srt/managers/schedule_batch.py

```diff
@@ -420,11 +435,15 @@ def batch_sampling_params(self, vocab_size, int_token_logit_bias):
             if reqs[i].sampling_params.dtype == "int":
                 if self.logit_bias is None:
                     self.logit_bias = torch.zeros(
-                        (bs, vocab_size), dtype=torch.float32, device=device
+                        (bs, self.model_config.vocab_size),
+                        dtype=torch.float32,
+                        device=device,
                     )
-                self.logit_bias[i][: len(int_token_logit_bias)] = int_token_logit_bias
+                self.logit_bias[i][
+                    : len(self.int_token_logit_bias)
+                ] = self.int_token_logit_bias
```

Comment:
We can remove `int_token_logit_bias` now. It was used before we had regex decoding. But please keep `logit_bias` because it is still useful and it is in OpenAI API (although we haven't supported it yet)